cli/pkg/backend/aws/k8s_compute_backend.go

```diff
@@ -281,3 +287,70 @@ func (k8s *K8SComputeBackend) Shell(ctx context.Context, stackName string, servi
 	streamOptions.TerminalSizeQueue = t.MonitorSize(t.GetSize())
 	return t.Safe(func() error { return exec.Stream(streamOptions) })
 }
+
+func (k8s *K8SComputeBackend) GetEvents(ctx context.Context, stackName string, services []string) error {
+	if len(services) == 0 {
+		return nil
+	}
+
+	for _, serviceName := range services {
+		deploymentName := fmt.Sprintf("%s-%s", stackName, serviceName)
```

Comment:
Might be good to create an abstraction for deploymentName since it's used in a number of funcs in this file

Reply:
Maybe the whole instantiation of the `pods` variable should be abstracted since that's where deploymentName is used